cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -229,6 +230,9 @@ func getRawEtcdEndpointsFromPodAnnotationWithoutRetry(client clientset.Interface
 	}
 	etcdEndpoints := []string{}
 	for _, pod := range podList.Items {
+		if !v1pod.IsPodReady(&pod) {
+			klog.V(3).Infof("pod %q is not ready", pod.ObjectMeta.Name)
+		}
```

Comment:
we cannot import k8s.io/kubernetes/pkg in kubeadm
you can try printing the status another way.

Reply:
https://github.com/kubernetes/kubernetes/blob/237d3dfda70d898e8b7641a295f5bdfecd527c2e/staging/src/k8s.io/kubectl/pkg/util/podutils/podutils.go#L186

Reply:
@neolit123  thanks so much.